lib/Backends/Interpreter/Interpreter.cpp

```diff
@@ -145,10 +145,12 @@ bool Interpreter::isOpSupported(const NodeInfo &NI) const {
            (NI.getOutElemTy(MaxPoolNode::ArgmaxIdx) == ElemKind::Int64ITy);
 
   case Kinded::Kind::ArgMaxNodeKind:
+  case Kinded::Kind::ArgMinNodeKind:
     return NI.allInputsAndOutputsHaveSameElemKind(
                {ElemKind::FloatTy, ElemKind::Int8QTy}, {},
-               {ArgMaxNode::ArgmaxIdx}) &&
-           (NI.getOutElemTy(ArgMaxNode::ArgmaxIdx) == ElemKind::Int64ITy);
+               {ArgMaxNode::ResultIdx}) &&
+           (NI.getOutElemTy(ArgMaxNode::ResultIdx) == ElemKind::Int64ITy ||
+            NI.getOutElemTy(ArgMaxNode::ResultIdx) == ElemKind::Int32ITy);
```

Comment:
Well I don't think it is a problem because all the nodes are similar and they have only one output (ResultIdx = 0). It would make no sense for any of them to diverge from the rest.

Reply:
Ok I guess that seems fine